test/test_mkldnn.py

```diff
@@ -309,6 +309,12 @@ def test_conv2d_legacy_jit_model(self):
                 conv2d(x),
                 conv2d_loaded(x.to_mkldnn()).to_dense())
 
+    def test_conv1d_functional(self):
```

Comment:
Please add a comment explaining why this test is here, maybe linking back to the issue.

I also suggest verifying the output shape as an additional sanity check.

Reply:
Done.